sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1191,7 +1191,7 @@ class DataSourceV2SQLSuite
   }
 
   test("tableCreation: duplicate column names in the table definition") {
-    val errorMsg = "Found duplicate column(s) in the table definition of `t`"
+    val errorMsg = "Found duplicate column(s) in the table definition of t"
```

Comment:
Only `IdentifierImpl` always adds the outer quote symbol. I make it consistent: https://github.com/apache/spark/pull/26817/files#diff-4885a1f40c0c8766af2cc33104a4c8e8R56

Reply:
I did this change because I added `quoteNameParts` to remove code duplication and then found `IdentifierImpl` use a slightly different quoting implementation.

Reply:
+1 for not using quotes unless it is necessary.